cmd/webhook/main.go

```diff
@@ -247,26 +241,23 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 			// eventing
 			eventingv1beta1.Kind("Trigger"): {
```

Comment:
should we just comment these out? There will be no conversion. Maybe it is a no-op?